doc/doxygen/install/install-linux.doxygen

```diff
@@ -1,72 +1,65 @@
 /**
 
-  @page install_linux Building %OpenMS on GNU/Linux
+  @page install_linux Building OpenMS on GNU/Linux
```

Comment:
yeah, the other way 'round of course. Dunno about comments, but it's removing the namespace link, but oxygen does so automatically in ```@page``` but not ```@section```. Also ```@page``` still shows %OpenMS in the browser tab and browser page title - at least my doxygen does so :(